### PR TITLE
fix subprocess not stopping bug when main process was killed

### PR DESCRIPTION
**Pull request by SunGaofeng**

### CODE REVIEW

File: PaddleCV/video/utils.py [hunk 1]
```diff
@@ -12,9 +12,20 @@
 #See the License for the specific language governing permissions and
 #limitations under the License.
 
+import os
+import signal
+
 __all__ = ['AttrDict']
 
 
+def term(sig_num, addition):
```
Comment (heavengate) [inline PaddleCV/video/utils.py]: term -> _term

File: PaddleCV/video/utils.py [hunk 2]
```diff
@@ -12,9 +12,20 @@
...
+    print('current pid is %s, group id is %s' % (os.getpid(), os.getpgrp()))
+    os.killpg(os.getpgid(os.getpid()), signal.SIGKILL)
```
Comment (heavengate) [inline PaddleCV/video/utils.py:23]: 是不是用SIGTERM比较好？

Comment (SunGaofeng) [reply PaddleCV/video/utils.py:23]: SIGTERM会造成term函数反复被调用，SIGKILL可以直接把进程组里的所有进程全部杀掉

**Review by heavengate — APPROVED** [review]
LGTM

File: PaddleCV/video/utils.py (hunk 1 above)
Comment (SunGaofeng) [reply PaddleCV/video/utils.py]: done

**Comment by heavengate** [comment]
fix https://github.com/PaddlePaddle/models/issues/2170

**Pull request merged**